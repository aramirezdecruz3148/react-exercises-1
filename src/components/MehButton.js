import React, { useState } from 'react';

function MehButton() {
  const buttonStyles = {
    marginRight: '10px',
    marginLeft: '10px',
    background: '#EE82EE',
    color: 'white'
  };

  const tallyStyles = {
    color: '#32CD32'
  };

  const [count, setCount] = useState(0);

  return (
    <>
    <button onClick={() => setCount(count + 1)} style={buttonStyles}>Meh, she is okay...</button>
    <span style={tallyStyles}>Meh Tally: {count}</span>
    </>
  );
}

export default MehButton;
