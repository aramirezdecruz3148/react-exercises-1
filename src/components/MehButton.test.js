import React from 'react';
import { shallow } from 'enzyme';
import MehButton from './MehButton';

describe('MehButton component', () => {
  it('renders MehButton', () => {
    const wrapper = shallow(<MehButton />);
    expect(wrapper).toMatchSnapshot();
  });
});
