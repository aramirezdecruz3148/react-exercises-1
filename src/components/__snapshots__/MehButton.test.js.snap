// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MehButton component renders MehButton 1`] = `
<Fragment>
  <button
    onClick={[Function]}
    style={
      Object {
        "background": "#EE82EE",
        "color": "white",
        "marginLeft": "10px",
        "marginRight": "10px",
      }
    }
  >
    Meh, she is okay...
  </button>
  <span
    style={
      Object {
        "color": "#32CD32",
      }
    }
  >
    Meh Tally: 
    0
  </span>
</Fragment>
`;
